### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by Divyanshu9794**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -220,6 +220,7 @@
 * [Android Developer Fundamentals (Version 2) — Codelab](https://developer.android.com/courses/fundamentals-training/toc-v2)
 * [Android Developer Fundamentals (Version 2) — Concepts](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/index.html)
 * [Android Development for Beginners - Full Course](https://www.youtube.com/watch?v=fis26HvvDII) - freeCodeCamp.org
+* [Android Development Full Tutorial 2023 | Kotlin | Complete Course with Roadmap & Projects | ‪@SCALER‬](https://www.youtube.com/watch?v=9-pFPGAOSZQ) - Scaler
```
Comment (Ajay-singh1) [inline courses/free-courses-en.md:223]:
```suggestion
* [Android Development Full Tutorial 2023 \| Kotlin \| Complete Course with Roadmap & Projects \| ‪@SCALER‬](https://www.youtube.com/watch?v=9-pFPGAOSZQ) - Scaler
```

**Review by Ajay-singh1 — COMMENTED** [review]
Put a \ before |

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [inline courses/free-courses-en.md:223]:
```suggestion
* [Android Development Full Tutorial 2023 \| Kotlin‬](https://www.youtube.com/watch?v=9-pFPGAOSZQ) - Scaler
```
But also, please move this to the Kotlin section

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1236,6 +1236,7 @@
 ### Kotlin
 
 * [Advanced Android with Kotlin](https://www.udacity.com/course/advanced-android-with-kotlin--ud940) (Udacity)
+* [Android Development Full Tutorial 2023 \| Kotlin‬](https://www.youtube.com/watch?v=9-pFPGAOSZQ) - Scaler
```
Comment (eshellman) [inline courses/free-courses-en.md:1239]:
```suggestion
* [Android Development Full Tutorial 2023 \| Kotlin](https://www.youtube.com/watch?v=9-pFPGAOSZQ) - Scaler
```

**Pull request merged**